changelog/changelog2022.md

```diff
@@ -0,0 +1,11 @@
+# Changelog 2022
+
+## 19th September 2022
+
+* [Sources](../mews-operations/reservations.md#source) replaces Channel in the [Process Group Request](../mews-operations/reservations.md#request), which now supports processing of multiple sources.
+* [Travel Agency](../mews-operations/reservations.md#travel-agency) was added to [Process Group Request](../mews-operations/reservations.md#request).
```

Comment:
I would say "In the [Process Group](../mews-operations/reservations.md#process-group) request message, `company` is split into separate entities for `company` and `travelAgency`. `iata` is removed from `company` (deprecated). [`travelAgency`](../mews-operations/reservations.md#travel-agency) is a new object."